.github/workflows/kind-e2e.yaml

```diff
@@ -17,8 +17,8 @@ jobs:
       fail-fast: false # Keep running if one leg fails.
       matrix:
         # Keep in sync with the list of supported releases: https://kubernetes.io/releases/
+        # TODO: add 1.31.x once it is added in https://github.com/sigstore/scaffolding/releases
```

Comment:
How do I check which versions are supported?

Reply:
Github action uses this [script](https://github.com/sigstore/scaffolding/blob/main/hack/setup-kind.sh#L68-L94) to install kind cluster and if fetches from the version specified here https://github.com/tektoncd/chains/blob/main/.github/workflows/reusable-e2e.yaml#L30
And here I did not see 1.31.x

Reply:
Cool. It would be great to add a little comment here pointing to the script.